dbms/src/Interpreters/ExternalModels.h

```diff
@@ -1,59 +0,0 @@
-// Copyright 2022 PingCAP, Ltd.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-#pragma once
-
-#include <Dictionaries/CatBoostModel.h>
```

Comment:
Can we also remove the `CatBoostModel`?

Reply:
> Can we also remove the `CatBoostModel`?

Yes, I have another pr that removed some models.